packages/example-app-base/src/components/ExampleCard/ExampleCard.tsx

```diff
@@ -37,20 +39,23 @@ export class ExampleCardBase extends React.Component<IExampleCardProps, IExample
   private _themeCustomizations: IExampleCardCustomizations[] | undefined;
   private _themeOptions: IDropdownOption[];
   private _classNames: IProcessedStyleSet<IExampleCardStyles>;
+  private readonly canRenderLiveEditor: boolean;
 
   constructor(props: IExampleCardProps) {
     super(props);
 
     this.state = {
-      isCodeVisible: false,
       schemeIndex: 0,
       themeIndex: 0
     };
+
+    this.canRenderLiveEditor =
+      getSetting('useEditor') === '1' && !isIE11() && transformExample(props.code!, 'placeholder').error === undefined ? true : false;
```

Comment:
Typically we wouldn't include the explicit `? true : false`.
```suggestion
      getSetting('useEditor') === '1' && !isIE11() && transformExample(props.code!, 'placeholder').error === undefined;
```